cassandra/cqlengine/management.py

```diff
@@ -119,9 +119,9 @@ def _get_index_name_by_column(table, column_name):
             return index_metadata.name
 
 
-def sync_table(model):
+def sync_table(model, keyspaces=None):
     """
-    Inspects the model and creates / updates the corresponding table and columns.
+    Inspects the model and creates / updates the corresponding table and columns for all keyspaces.
```

Comment:
For all keyspaces specified. umm... should we automatically include the default Class.__keyspace__ in the keyspaces list? it is not at the moment.